compiler/damlc/tests/src/DA/Test/DamlcIntegration.hs

```diff
@@ -172,10 +172,11 @@ getIntegrationTests registerTODO scenarioService = do
     putStrLn $ "rtsSupportsBoundThreads: " ++ show rtsSupportsBoundThreads
     do n <- getNumCapabilities; putStrLn $ "getNumCapabilities: " ++ show n
 
-    damlTests <- getDamlTestFiles "compiler/damlc/tests/daml-test-files"
-    bondTradingTests <- getBondTradingTestFiles
-    let plainTests = damlTests <> bondTradingTests
-    noScenariosEnabledTests <- getDamlTestFiles "compiler/damlc/tests/daml-test-files/no-scenarios-enabled"
+    plainTests <- getDamlTestFiles "compiler/damlc/tests/daml-test-files"
```

Comment:
I also flip the tests. Now, `plainTests` refers to the tests that run with `--enable-scenarios=no`, and `scenariosEnabledTests` (including the bond trading test) are the "marked" case, running with the non-default `--enable-scenarios=yes`. I guess with time we should try to move tests from the latter to the former, or at least avoid adding new tests to the latter.